R/render-plot.R

```diff
@@ -272,6 +275,7 @@ drawPlot <- function(name, session, func, width, height, pixelratio, res, ...) {
         src = session$fileUrl(name, outfile, contentType='image/png'),
         width = width,
         height = height,
+        alt = alt,
```

Comment:
Similar to `result$coordmap`, we need to store the posibility of `alt` being returned from the function above (where `coordmap` is created).

In that return value, we would add `alt` if `value` inherits from `shiny_rendered_plot`. This will be attached to the result object which will be passed around, making the extra `alt` arguments unnecessary.

Reply:
This would be the location to default `alt` to `"Plot object"`

Reply:
This location may be too late to capture the `alt` value.  At this point, the print method has been called on the plot object.  Instead, we'd like to capture the alt before it's printed.

May have to add a check in-between :

```r
          function(value, .visible) {
            if (.visible) {
```

... in the `hybrid_chain` above.

This way, we can always check for the `alt` information even if nothing is visible (such as with `base` graphics).

Reply:
(not true anymore. Resolving)